recipes/enaml/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.10.1" %}
+package:
+  name: enaml
+  version: {{ version }}
+
+source:
+  url: https://github.com/nucleic/enaml/archive/{{ version }}.tar.gz
+  fn: enaml-{{ version }}.tar.gz
+  sha256: f6500bda40cdff006b4dc0ea7f4a0ae820542b2fbae3362f48dcc2194d84dfcc
+
+build:
+  number: 0
+  entry_points:
+    - enaml-run = enaml.runner:main
+  osx_is_app: True
+  skip_compile_pyc:
+    - "*/enaml/core/byteplay/*.py"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - atom >=0.4.0
+    - kiwisolver >=0.2.0
+    - ply
+    - future
+    - qtpy >=1.3
+  run:
+    - python
+    - python.app  # [osx]
+    - atom >=0.4.0
+    - pyqt
+    - qtpy >=1.3
+    - kiwisolver >=0.2.0
+    - ply
+    - future
+
+test:
+  commands:
+    - enaml-run --help
+  imports:
+    - enaml
+    - enaml.callableref
+    - enaml.colorext
+    - enaml.core.alias
+    - enaml.core.dynamicscope
+    - enaml.core.funchelper
+    - enaml.fontext
+    - enaml.signaling
+    - enaml.weakmethod
+    - enaml.core.api
+    - enaml.widgets.api
+    - enaml.layout.api
+
+about:
+  home: https://github.com/nucleic/enaml
+  license: BSD
+  license_file: COPYING.txt
+  summary: Declarative DSL for building rich user interfaces in Python
+
+extra:
+  recipe-maintainers:
+    - blink1073
```

Comment:
Can you please check with the other maintainers (it is find to just @ mention them in this PR and ask for them to confirm their willingness)

Reply:
Alternatively, you could just list yourself as the maintainer for now.